src/Sulu/Bundle/MediaBundle/Resources/js/containers/MediaCollection/CollectionSection.js

```diff
@@ -113,14 +113,15 @@ class CollectionSection extends React.Component<Props> {
     };
 
     handleSaveResponse = (resourceStore: ResourceStore) => {
+        this.closeCollectionOperationOverlay();
```

Comment:
the `this.props.onCollectionNavigate(resourceStore.id)` call below will clear the data of the `CollectionFormOverlay`. we should close the overlay before the data is cleared, otherwise the user sees that the input fields are cleared before the overlay is fully closed 🙂